### PR TITLE
Fix for NODE_UPGRADE txn

### PR DESCRIPTION
**Pull request by ArtObr**
Signed-off-by: ArtObr <artemobruchnikov@gmail.com>

### CODE REVIEW

File: indy_node/test/did/test_did_with_abbreviated_verkey.py [hunk 1]
```diff
@@ -1,38 +1,28 @@
...
+
+from indy_common.constants import NODE_UPGRADE
+from plenum.test.helper import sdk_get_and_check_replies
+from plenum.test.pool_transactions.helper import sdk_sign_and_send_prepared_request
+
+from plenum.common.constants import TXN_TYPE, CURRENT_PROTOCOL_VERSION
+from plenum.common.types import OPERATION, f
+
+
+def test_send_audit_txn1(looper, sdk_wallet_client, sdk_pool_handle):
```
Comment (skhoroshavin) [inline indy_node/test/did/test_did_with_abbreviated_verkey.py]: Why is it called send audit?

File: indy_node/test/did/test_did_with_abbreviated_verkey.py [hunk 2]
```diff
@@ -1,38 +1,28 @@
...
-def testRetrieveChangedVerkey(looper, tconf, nodeSet, sdk_pool_handle, sdk_wallet_trustee, nym_abbrv_vk):
-    wh, did, vk = nym_abbrv_vk
-    new_vk = sdk_rotate_verkey(looper, sdk_pool_handle, wh, did, did)
-    resp_data = nym_get(looper, sdk_pool_handle, sdk_wallet_trustee, did)
-    assert did == resp_data[0]
-    assert vk != resp_data[1]
-    assert new_vk == resp_data[1]
-
-
-def testVerifySigWithChangedVerkey(looper, tconf, nodeSet, sdk_pool_handle, nym_abbrv_vk):
```
Comment (skhoroshavin) [inline indy_node/test/did/test_did_with_abbreviated_verkey.py]: Why old tests are removed?

File: indy_node/test/did/test_did_with_abbreviated_verkey.py (hunk 1 above)
Comment (ArtObr) [reply indy_node/test/did/test_did_with_abbreviated_verkey.py]: fixed

File: indy_node/test/did/test_did_with_abbreviated_verkey.py (hunk 2 above)
Comment (ArtObr) [reply indy_node/test/did/test_did_with_abbreviated_verkey.py]: fixed

**Comment by ArtObr** [comment]
https://jira.hyperledger.org/browse/INDY-2158

**Pull request merged**